include/seqan/align/dp_formula_affine.h

```diff
@@ -59,30 +59,31 @@ namespace seqan {
 // Function _internalComputeScore      [RecursionDirectionDiagonal, AffineGaps]
 // ----------------------------------------------------------------------------
 
-template <typename TScoreValue, typename TAffineGaps, typename TTraceValueL, typename TTraceValueGap>
-inline SEQAN_FUNC_ENABLE_IF(Not<Is<SimdVectorConcept<TScoreValue> > >, typename TraceBitMap_<TScoreValue>::Type)
-_internalComputeScore(DPCell_<TScoreValue, TAffineGaps> & activeCell,
+template <typename TScoreValue, typename TTraceValueL, typename TTraceValueGap>
+inline SEQAN_FUNC_ENABLE_IF(Not<Is<SimdVectorConcept<TScoreValue> > >,
+                            typename TraceBitMap_<TScoreValue>::Type)
+_internalComputeScore(DPCell_<TScoreValue, AffineGaps> & activeCell,
                       TScoreValue const & rightCompare,
                       TTraceValueL,
                       TTraceValueGap,
                       TracebackOff const &,
                       RecursionDirectionDiagonal const &)
 {
-    if(activeCell._score < rightCompare)
-        activeCell._score = rightCompare;
+    using std::max;
+    _scoreOfCell(activeCell) = max(_scoreOfCell(activeCell), rightCompare);
     return TraceBitMap_<TScoreValue>::NONE;
 }
 
-template <typename TScoreValue, typename TAffineGaps, typename TTraceValueL, typename TTraceValueGap>
+template <typename TScoreValue, typename TTraceValueL, typename TTraceValueGap>
 inline SEQAN_FUNC_ENABLE_IF(Is<SimdVectorConcept<TScoreValue> >, typename TraceBitMap_<TScoreValue>::Type)
-_internalComputeScore(DPCell_<TScoreValue, TAffineGaps> & activeCell,
+_internalComputeScore(DPCell_<TScoreValue, AffineGaps> & activeCell,
```

Comment:
is the rest also wrong then?
see e.g. line 91 (same file) also stating `TAffineGaps` and not `AffineGaps`.